src/lib/components/input/input.js

```diff
@@ -1,83 +1,77 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
-import { componentPropTypes, defaultComponentPropTypes } from '../../utils/componentPropTypes';
-import { withValidation } from '../../utils/hocs';
```

Comment:
… and we will need this to make the validations work …

Reply:
I've just changed it so that we're pulling the file from the `input` folder rather than the `oldInput` folder which will soon be deprecated. I hope that's ok @rin